packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -283,7 +283,7 @@ export default decorate([
           id: props.job.id,
           name: state.name,
           mode: state.isDelta ? 'delta' : 'full',
-          compression: state.compression,
+          compression: state.isFull ? state.compression : '',
```

Comment:
I kind of like the idea to keep settings set even if unused for a given type so that if the type of the job change again, it will revert to the previous setting.

But I'm open to discussion on this.